docs/usage.md

```diff
@@ -0,0 +1,187 @@
+
+The `getSelf()` method can be used to return the `spaceMember` object for the local client.
+
+## Enter a space
+
+When a user enters a space they should call `enter()`. This publishes an event to all subscribers of that space.
+
+`space.enter()` can take an optional object called `profileData` so that users can include convenient metadata to update an avatar stack, such as a username and profile picture.
+
+The following is an example of entering a space with `profileData`:
+
+```ts
+space.enter({
+  username: "Claire Lemons",
+  avatar: "https://slides-internal.com/users/clemons.png",
+});
+```
+
+### Leave a space
+
+A leave event is sent when a user leaves a space. This can occur for one of the following reasons:
+
+* `space.leave()` is called explicitly.
+* The user closes the tab.
+* The user is abruptly disconnected from the internet for longer than 2 minutes.
+  * Note that the time before they are seen has having left the space is configurable using `offlineTimeout`.
+
+## Subscribe to location updates
+
+Subscribe to `locationUpdate` events in order to display where users are within a space, such as which slide number they are currently viewing, or which cell or component they have selected. `locationUpdate` events for a space are sent when a user calls `locations.set()`, which can be configured to fire for whichever UI interactions you'd like. Use the `locations.on()` method to register a listener for `locationUpdates` events.
+
+The following is an example of subscribing to location updates for a space:
+
+```ts
+space.locations.on('locationUpdate', (update) => {
+  console.log(update);
+});
+```
+
+The following is an example `locationUpdate` event received by subscribers when a user changes location:
+
+```json
+{
+    "member": {
+    "clientId": "clemons@slides.com",
+    "isConnected": true,
+    "profileData": {
+      "username": "Claire Lemons",
+      "avatar": "https://slides-internal.com/users/clemons.png"
+    },
+    "location": {"slide": "3", "component": "slide-title"},
+    "lastEvent": { "name": "update", "timestamp": 1 },
+    "connections": ["2"],
+  },
+  "previousLocation": {"slide": "2", "component": null },
+  "currentLocation": {"slide": "3", "component": "slide-title"}
+}
+```
+
+The following are the properties of a `locationUpdate` event:
+
+| Property | Description | Type |
+| -------- | ----------- | ---- |
+| member | The details of the user that has changed location. | [`spaceMember`](#subscribe-to-member-updates) |
+| previousLocation | The previous location of the user within the space. | any |
+| currentLocation | The current location of the user within the space. | any |
+
+To stop subscribing to location updates, users can call the `locations.off()` method.
+
+### Track an individual location or user
+
+It is also possible to track a specific location or user, rather than subscribing to all events using `createTracker()`.
```

Comment:
Worth adding a note here that they'd still receive all the updates but using the tracker, they can filter out those updates for a specific location.